src/tir/transforms/inject_ptx_async_copy.cc

```diff
@@ -79,8 +80,10 @@ class PTXAsyncCopyInjector : public StmtMutator {
         if (indices_lanes == 1) {
           auto src_offset = load->indices[0];
           auto dst_offset = store->indices[0];
-          Array<PrimExpr> args = {store->buffer->data, tir::Mul(dst_offset, PrimExpr(index_factor)),
```

Comment:
We can use helper function `TVM_DLL PrimExpr mul(PrimExpr a, PrimExpr b, Span span = Span()` to do automatical integer type upcasting.

Reply:
Thanks for the advise! I've changed it by calling `TVM_DLL PrimExpr mul(PrimExpr a, PrimExpr b, Span span = Span()` .
Please check again.